api.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const api = axios.create({
+    baseURL: "https://nc-news-ch8f.onrender.com/api"
+});
+
+export default api;
```

Comment:
Nice, but this is also where we should be using our api instead of exporting it.